options/locale/locale_en-US.ini

```diff
@@ -1026,10 +1026,8 @@ transfer.no_permission_to_reject = You do not have permission to reject this tra
 
 desc.private = Private
 desc.public = Public
-desc.private_template = Private template
-desc.public_template = Template
+desc.template = Template
```

Comment:
this will mess with backportin translations  :/

Reply:
That's exactly the reason I'm backporting this PR - so that we can safely backport translations without breaking anything

Reply:
Otherwise backporting translations will result in nonexistent translation strings inside the template

Reply:
well if you will manually packport the upcomming translation ... I wont block